FHEM/lib/SD_ProtocolData.pm

```diff
@@ -22,9 +22,9 @@
 #
 # length_min  => ' '       # minimum number of bits of message length
 # length_max  => ' '       # maximum number of bits of message length
-# paddingbits => ' '       # pad up to x bits before call module, default is 4.
-# paddingbits => '1'       # will disable padding, use this setting when using dispatchBin
-# paddingbits => '2'       # is padded to an even number, that is a maximum of 1 bit
+# paddingbits => ' '       # only MS - pad up to x bits before call module, default is 4.
```

Comment:
paddingbits wird auch bei MU Verarbeitung berücksichtigt.

Reply:
wird es nur noch bei MU gemacht? oder auch MC?

Reply:
Zusatzfrage: ist es gewollt, wenn paddingbits nicht in der Definition drin steht, das es IMMER aktiv ist?

Reply:
Paddingbits wird aber dann offensichtlich bei MU anders gehandhabt als bei MS.
Uns ist es aufgefallen, als wir die Definition 74.1 ergänzt haben, das bei der 74.1 bei identischen Nachrichten 60 Bit an "postDemodulation => \&main::SIGNALduino_postDemo_FS20," übergeben werden, aber bei Definition 74 (MU)  nur 58 Bit.

Vieleicht ist auch nur die Reihenfolge verschieden:
MU: postDemodulation, paddingbits
MS: paddingbits, postDemodulation
Es wäre aber schon besser, wenn das einheitlich wäre.

Reply:
Der Standard ist bei beiden Identisch, es wir auf volle Nibble aufgefüllt:

https://github.com/RFD-FHEM/RFFHEM/blob/556dbc1a4a9181aeb07a332cf4ae0f00394f7279/FHEM/00_SIGNALduino.pm#L2150

MU:
https://github.com/RFD-FHEM/RFFHEM/blob/556dbc1a4a9181aeb07a332cf4ae0f00394f7279/FHEM/00_SIGNALduino.pm#L2426

Lediglich die Implementierung könnte ich mal angleichen :)